src/bundle/Resources/public/js/scripts/admin.notifications.js

```diff
@@ -1,23 +1,29 @@
 (function(global, doc, eZ, bootstrap) {
-    const notificationsContainer = doc.querySelector('.ez-notifications-container');
+    const notificationsContainer = doc.querySelector('.ibexa-notifications-container');
     const notifications = JSON.parse(notificationsContainer.dataset.notifications);
     const template = notificationsContainer.dataset.template;
+    const iconsMap = {
+        info: 'system-information',
+        error: 'circle-close',
+        warning: 'warning-triangle',
+        success: 'checkmark',
+    };
     const addNotification = ({ detail }) => {
         const { onShow, label, message, rawPlaceholdersMap = {} } = detail;
-        const templateLabel = label === 'error' ? 'danger' : label;
         const config = eZ.adminUiConfig.notifications[label];
         const timeout = config ? config.timeout : 0;
         const container = doc.createElement('div');
+        const iconPath = eZ.helpers.icon.getIconPath(iconsMap[label]);
```

Comment:
Is there a possibility there will be some new type that will break this code?

And should we care about new notification types added by integrators at least by adding some default fallback?

Reply:
Also not breaking UI, just do not display at all. Maybe we could add possibility to add iconPath in the event details